content/en/tracing/setup/go.md

```diff
@@ -163,6 +163,28 @@ func main() {
 
 ```
 
+## Configure APM Environment Name
+
+Optionally configure the [APM environment name][54] when starting the tracer with [WithEnv][55].
+When configured, the tracer attaches an `env` tag to all traces.
+If you have configured an environment name [in the agent][56], the agent configuration is overridden.
+
+```go
+package main
+
+import (
+    "os"
+
+    "gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func main() {
+    tracer.Start(tracer.WithEnv("<ENVIRONMENT>"))
+    defer tracer.Stop()
```

Comment:
I would add two more lines here, otherwise this program looks weird:
```
func main() {
    tracer.Start(tracer.WithEnv("<ENVIRONMENT>"))
    defer tracer.Stop()
    
    // ...
}
```

Reply:
👍